python/triton/language/core.py

```diff
@@ -1215,7 +1215,24 @@ def max_contiguous(input, values, _builder=None):
 
 @triton.jit
 def abs(x):
```

Comment:
I think it's possible to have a single implementation for all floating point types using `dtype.primitive_bitwidth` https://github.com/openai/triton/blob/128ada30439d802ab7e3cfeb5c2db43030a688e9/python/triton/language/core.py#L64. Also, is `abs` currently tested in `test_core.py`? If not, maybe we should add a test for it

Reply:
just did that